frontend/src/Components/Helpers/Helpers.js

```diff
@@ -0,0 +1,96 @@
+// Helper functions
+import axios from "axios";
+
+export function processResults(resultArr, isCBA) {
+    const data_to_visualize_all = [];
+    let isTimeSeries = false;
+    if (!resultArr) {
+        resultArr = [];
+        isTimeSeries = false;
+    } else if (resultArr[0][0] !== undefined && resultArr[0][0].length === 0) {
+        isTimeSeries = false;
+    } else if (resultArr[0][Object.keys(resultArr[0])[0]][0].time) {
+        isTimeSeries = true;
+    } else {
+        isTimeSeries = false;
+    }
```

Comment:
Extraneous conditionals. `isTimeSeries` is already set to `false`. No need to re-set it